.githooks/pre-push

```diff
@@ -1,5 +1,19 @@
 #!/bin/sh
```

Comment:
This is what chatGPT says:

The Apache license header can be placed before or after the shebang. However, it's generally recommended to put the shebang as the first line of the script so that the operating system knows which interpreter to use to execute the script. So, in the given example, it is appropriate to have the shebang as the first line, followed by the Apache license header.